### PR TITLE
[MH2018-22] Sh feature/product info

### PR DESCRIPTION
**Pull request by IsmaelVillavicencio**
Originals
desktop 1024
mobile portrait

How it looks!!
https://fiorella-dev.myshopify.com/collections/frontpage/products/flower-bouquet-victoria

### CODE REVIEW

File: fiorella/src/styles/layout/_product.scss [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+  label {
+    border-radius: 50%;
+    float: left;
+    height: 40px;
+    width: 40px;
+  }
+
+  .swatch-element {
+    margin:10px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Colon after property should be followed by one space

File: fiorella/src/styles/layout/_product.scss [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+}
+.swatch {
+  display: flex;
+  flex-direction: row;
+
+  input {
+    display: none;
+
+    &:checked + label {
+      box-shadow:0px 0px 10px $gray-8;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Colon after property should be followed by one space<br>`0px` should be written without units as `0`

File: fiorella/src/styles/layout/_product.scss [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  }
+
+  .swatch-element {
+    margin:10px;
+
+    label {
+      padding: 0 10px;
+    }
+
+    .crossed-out {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    border-radius: 50%;
+    float: left;
+    height: 40px;
+    width: 40px;
+  }
+
+  .swatch-element {
+    margin:10px;
+
+    label {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+  .social-sharing {
+    display: flex;
+    justify-content: space-around;
+
+    a {
+      color: $cerise-red;
+      text-decoration: none;
+
+      &:hover {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+      margin-bottom: 28px;
+      margin-top: 28px;
+    }
+  }
+
+  .social-sharing {
+    display: flex;
+    justify-content: space-around;
+
+    a {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+
+    .product-title {
+      display: none;
+      font-family: $playfair-display;
+      @media #{ $tablet }, #{ $desktop } {
+        display: block;
+      }
+    }
+
+    .product-description {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+
+  .product-img-box {
+    width: 100%;
+  }
+
+  .product-info-box {
+    padding: 20px;
+    width: 100%;
+
+    .product-title {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/layout/_product.scss [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+}
+
+.product-color-box {
+  display: flex;
+  justify-content: center;
+
+  @media #{ $tablet }, #{ $desktop } {
+    padding-left: 30px;
+  }
+}
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Rule declaration should be followed by an empty line

File: fiorella/src/styles/layout/_product.scss [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+      width: 220px;
+    }
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    flex-direction: row;
+    margin: 0;
+
+  }
+}
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Rule declaration should be followed by an empty line

File: fiorella/src/styles/layout/_product.scss [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+      margin-top: 28px;
+      width: 220px;
+    }
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    flex-direction: row;
+    margin: 0;
+  }
+}
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Rule declaration should be followed by an empty line

File: fiorella/src/styles/layout/_product.scss [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+    color: $cerise-red;
+    font-family: $merriweathersans;
+    font-size: 30px;
+    font-weight: bold;
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    margin: 20px 0;
+  }
+}
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Rule declaration should be followed by an empty line

File: fiorella/src/styles/layout/_product.scss [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+    }
+  }
+}
+
+.quantity-box {
+  font-family: $merriweathersans;
+  font-weight: bold;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  margin-top: 0;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Property `margin-top` already defined on line 48

File: fiorella/src/styles/layout/_product.scss [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+  @media #{ $tablet }, #{ $desktop } {
+    margin: 20px 0;
+  }
+}
+.btn-submit {
+  margin: 20px 0;
+  text-align: center;
+
+  .btn-add-to-car {
+    background-color: $cerise-red;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

File: fiorella/src/styles/layout/_product.scss [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+    border-radius: 50%;
+    float: left;
+    height: 40px;
+    width: 40px;
+  }
+
+  .swatch-element {
+    margin: 10px;
+
+    label {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/sections/product.liquid [hunk 16]
```diff
@@ -131,3 +124,11 @@
     ]
   }
 {% endschema %}
+
+<style type="text/css">
+#Quantity{
```
Comment (luisarmandodv) [inline fiorella/src/sections/product.liquid]: Identación

File: fiorella/src/sections/product.liquid [hunk 17]
```diff
@@ -131,3 +124,11 @@
...
+  width: 50px;
```
Comment (luisarmandodv) [inline fiorella/src/sections/product.liquid]: La propiedad width debe ir al final

File: fiorella/src/sections/product.liquid [hunk 18]
```diff
@@ -67,50 +60,50 @@
...
-          <span class="visually-hidden" data-compare-text>{{ 'products.product.regular_price' | t }}</span>
-          <s data-compare-price>
-            {% if current_variant.compare_at_price > current_variant.price %}
-              {{ current_variant.compare_at_price | money }}
-            {% endif %}
-          </s>
-        {% endif %}
+      <div class="quantity-box">
+        <label for="Quantity">{{ 'products.product.quantity' | t  }}</label>
+        <input type="number" id="Quantity" name="quantity" value="1" min="1" >
```
Comment (luisarmandodv) [inline fiorella/src/sections/product.liquid]: Duda ¿No deberias mas bien usar un select? 🤔

File: fiorella/src/snippets/social-sharing.liquid [hunk 19]
```diff
@@ -3,24 +3,27 @@
 
   {% if settings.share_facebook %}
     <a target="_blank" href="//www.facebook.com/sharer.php?u={{ shop.url | append: share_permalink }}">
-      {% include 'icon-facebook' %}
-      <span aria-hidden="true">{{ 'general.social.share_on_facebook' | t }}</span>
+      {% include 'icon-facebook-share' %}
```
Comment (luisarmandodv) [inline fiorella/src/snippets/social-sharing.liquid]: ### Una duda, ¿No seria mas correcto utilizar el filtro asset_url? 

image

File: fiorella/src/snippets/swatch.liquid [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+  {% assign values = '' %}
+  {% for variant in product.variants %}
+    {% assign value = variant.options[option_index] %}
+    {% unless values contains value %}
+      {% assign values = values | join: ',' %}
+      {% assign values = values | append: ',' | append: value %}
+      {% assign values = values | split: ',' %}
+      <div data-value="{{ value | escape }}" class="swatch-element {% if is_color %}color {% endif %}{{ value | handle }} {% if variant.available %}available{% else %}soldout{% endif %}">
+        {% if is_color %}
+        {% comment %}
```
Comment (luisarmandodv) [inline fiorella/src/snippets/swatch.liquid]: ¿Es necesario este comentario?

File: fiorella/src/snippets/swatch.liquid [hunk 21]
```diff
@@ -0,0 +1,84 @@
...
+{% unless found_option %}
+<div class="swatch error">
+  <p>You included the snippet swatch.liquid with the name of a product option — <code>'{{ swatch }}'</code> — that does not belong to your product.</p>
+  <p>Use <code>{% raw %}{% include 'swatch' with 'name of your product option here' %}{% endraw %}</code></p>
+  <p>Example: <code>{% raw %}{% include 'swatch' with 'Color' %}{% endraw %}</code></p>
+  <p><strong>This is case-sensitive!</strong> Do not put in <code>'color'</code> if your product option name is <code>'Color'</code>.</p>
+</div>
+{% else %}
+<div class="swatch clearfix" data-option-index="{{ option_index }}">
+  {% comment %}
```
Comment (luisarmandodv) [inline fiorella/src/snippets/swatch.liquid]: ¿Es necesario este comentario?

File: fiorella/src/snippets/swatch.liquid [hunk 22]
```diff
@@ -0,0 +1,84 @@
+{% comment %}
```
Comment (luisarmandodv) [inline fiorella/src/snippets/swatch.liquid]: ¿Es necesario este comentario?

File: fiorella/src/sections/product.liquid (hunk 18 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/sections/product.liquid]: Eso lo da por default chopify... un select.. como se utilizaría?

File: fiorella/src/snippets/social-sharing.liquid (hunk 19 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/snippets/social-sharing.liquid]: No encontré la forma de acceder al icon que esta dentro de la carpeta icons... solo seguí la misma estructura que tenia ese archivo, ya esta por default.

File: fiorella/src/sections/product.liquid [hunk 23]
```diff
@@ -131,3 +124,11 @@
     ]
   }
 {% endschema %}
+
+<style type="text/css">
+  #Quantity{
```
Comment (jaimelr) [inline fiorella/src/sections/product.liquid]: There's a missing space before the curly brace and try not to use ids to target elements in CSS.

File: fiorella/src/styles/abstracts/_variables.scss [hunk 24]
```diff
@@ -12,6 +12,7 @@ $font-color-white: #ffffff;
 $border-color-white: #ffffff;
 $peppermind-color: #f3f9ef;
 $text-color-black: #000000;
+$gray-8: #cccccc;
```
Comment (jaimelr) [inline fiorella/src/styles/abstracts/_variables.scss]: You could use the shorthand `#ccc`

File: fiorella/src/sections/product.liquid [hunk 25]
```diff
@@ -131,3 +124,11 @@
     ]
   }
 {% endschema %}
+
+<style type="text/css">
+  #Quantity {
+    background: white url(https://image.flaticon.com/icons/svg/156/156320.svg) right no-repeat;
```
Comment (8geonirt) [inline fiorella/src/sections/product.liquid]: I suggest to download this asset because if that image is removed then the icon is not going to work anymore

Comment (IsmaelVillavicencio) [reply fiorella/src/sections/product.liquid]: You're right. :+1:

**Pull request merged**